pkg/cmd/roachtest/roachtestutil/mixedversion/steps.go

```diff
@@ -715,3 +778,58 @@ func startStopOpts(opts ...option.StartStopOption) []option.StartStopOption {
 		option.NoBackupSchedule,
 	}, opts...)
 }
+
+// TODO(kyleli): This step currently only affects the system tenant, should support panicking secondary tenants as well.
+// TODO(kyleli): Current mixedversion cannot support separating this into two steps (panic and restart) because during
+// each step the framework checks each node to ensure they are all alive and healthy. Ideally there should be a state
+// that allows the framework to skip this check, so that we can panic a node and then restart it in a separate step.
+type panicNodeStep struct {
+	initTarget int
+	targetNode option.NodeListOption
+	rt         test.Test
+}
+
+func (s panicNodeStep) Background() shouldStop { return nil }
+
+func (s panicNodeStep) Description() string {
+	return fmt.Sprintf("run panic node on node %d", s.targetNode[0])
```

Comment:
nit: lets mention that this is panicking the system interface, since it's not obvious without reading the code